java-engine/src/test/java/io/getunleash/engine/UnleashEngineTest.java

```diff
@@ -260,6 +261,7 @@ void customStrategiesRequired_whenNotConfigured_returnsFalse(
         assertEquals(expectedIsEnabled, result);
     }
 
+    @SuppressWarnings("unused")
```

Comment:
Not relevant to this PR but this warning was annoying me. The warning is raised because there's an `unused` PhantomReference in this test, which is absolutely important for the test to run